test/benchmark/instances.yaml

```diff
@@ -0,0 +1,4 @@
+test_vgg16_benchmark:
+  inf1.6xlarge:
+    instance_id: i-04b6adea9c066ad0f
+    key_filename: /Users/nikhilsk/nskool/serve/test/benchmark/ec2-key-name-6028.pem
```

Comment:
@nskool pls fix this line as in this test it was not required and key got generated and placed in fixed directory.

Reply:
Removed, and this file has been added to .gitignore.